ext/curl/tests/curl_unescape_basic001.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+curl_unescape basic test
+--SKIPIF--
+<?php if(!extension_loaded('curl')) print 'skip'; ?>
```

Comment:
The `curl_escape` and `curl_unescape` functions aren't necessarily going to be loaded (see [this related test for an example](https://github.com/php/php-src/blob/5f4a417537117ed5baadb4c0a1c738cdb4638c86/ext/curl/tests/curl_escape.phpt)).

Reply:
Also please provide a reason why the test has been skipped, such as "cURL extension not available".